update-requirements

```diff
@@ -146,11 +108,18 @@ def main():
     parser.add_argument('requirementsfile', help='Path to the requirements file',
                         type=argparse.FileType('r'))
     parser.add_argument('specfile', help='path to the specfile', type=argparse.FileType('r'))
+    parser.add_argument('--excludefile', help='Path/URL to the package-exclude.json (default: %(default)s)', default=PACKAGE_EXCLUDE_JSON_URL)
     args = parser.parse_args()
 
+    if args.excludefile.startswith('https://'):
+        excludes = requests.get(args.excludefile).json()
+    else:
+        with open(args.excludefile) as excludesfile:
+            excludes = json.load(excludesfile)
```

Comment:
I think it should be moved into the `if args.requirementstype == 'npm':` section so we don't reach out over the network for non-NPM.

Reply:
fair